app/views/step_by_step_pages/_form.html.erb

```diff
@@ -4,22 +4,7 @@
 %>
 
 <%= form_for(@step_by_step_page) do |form| %>
-  <% if step_by_step_page.errors.any? %>
-    <% error_explanation = capture do %>
-      <h2 class="h4">There are <%= pluralize(step_by_step_page.errors.count, "error") %></h2>
-
-      <ul>
-      <% step_by_step_page.errors.full_messages.each do |message| %>
-        <li><%= message %></li>
-      <% end %>
-      </ul>
-    <% end %>
-
-    <%= render "shared/steps/form_notice",
-      status: "danger",
-      message: error_explanation
-    %>
-  <% end %>
+  <%= render "shared/steps/form_errors", page: step_by_step_page %>
```

Comment:
I would call it a (maybe) a `resource` instead of a page as neither `step_by_step_page` and `step` are pages but models/resources.

Reply:
That makes sense. Renamed.